server/datastore/mysql/hosts.go

```diff
@@ -831,15 +831,6 @@ func (ds *Datastore) ListHosts(ctx context.Context, filter fleet.TeamFilter, opt
 		`
 	}
 
-	failingPoliciesSelect := `,
```

Comment:
Note: this function is also used by the Count Hosts endpoint, although I haven't (yet) found any use there of the failing policy membership join